VERSION

```diff
@@ -1,33 +1,6 @@
-v6.4.4
```

Comment:
Any reason why you had to remove all of these? When we migrated ark to `v3.0.0` we still had the previous release infos. 

If this is temporary add a TODO to re-add versions before release

Reply:
I did it because I wanted clean release notes (VERSION is used as the release notes at https://github.com/Clever/wag/releases); I'll add the older versions back under an "Older Versions" header.